src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$clockDiameter: 500px;
```

Comment:
The task description specifies that the size of the stopwatch should be 80vmin, but you have set a fixed pixel value. This might not make the stopwatch responsive. Consider using 'vmin' unit instead of 'px' for sizing.